dubbo-plugin/dubbo-compiler/src/main/java/org/apache/dubbo/gen/AbstractGenerator.java

```diff
@@ -360,6 +428,11 @@ private static class MethodContext {
         public String grpcCallsMethodName;
         public int methodNumber;
         public String javaDoc;
+        public HttpMethods httpMethod;
+        public String path;
+        public String body; // requestBody
```

Comment:
Make the comments readable
e.g.: Specify the message field that the HTTP request body mapping to